content.test.ts

```diff
@@ -0,0 +1,51 @@
+import fs from 'fs';
+import path from 'path';
+
+describe('Codecademy Docs Content', () => {
+  it('adheres to content file structure', () => {
+    // file names can only contain alphanumerics and hyphens
+    const validateName = (pathName: string, name: string) => {
+      const filteredName = name.replace(/[^A-Za-z0-9.-]/g, '');
```

Comment:
Can this happen with [String.match](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match)?  Also would https://jestjs.io/docs/expect#tomatchregexp--string be helpful?

Reply:
yeah I think `string.match` would make more sense here. I went with this kind of wonky `expect` pattern in order to make test failure messages more helpful (I wanted them all to include a path to the failing file), and I think doing a regex match might make that a little messier